NickvisionTagger.Shared/Controllers/MainWindowController.cs

```diff
@@ -312,6 +312,15 @@ public void UpdateTags(PropertyMap map, bool triggerSelectedMusicFilesProperties
                 pair.Value.Composer = map.Composer;
                 updated = true;
             }
+            if(map.BeatsPerMinute != pair.Value.BeatsPerMinute.ToString() && map.BeatsPerMinute != _("<keep>"))
+            {
+                try
+                {
+                    pair.Value.BeatsPerMinute = int.Parse(map.BeatsPerMinute);
```

Comment:
`uint.Parse`?

Reply:
With TagLib#, `year`, `track`, and `bpm` were `uint`s
With atldotnet they are `int`s